test-configs.yaml

```diff
@@ -176,12 +176,21 @@ test_plans:
         panfrost_prime
         panfrost_submit
 
-  kselftest:
+  kselftest: &kselftest
     rootfs: debian_buster_nfs
+    pattern: 'kselftest/{category}-{method}-{protocol}-{rootfs}-kselftest-template.jinja2'
     filters:
       - passlist: {defconfig: ['kselftest']}
     params:
       job_timeout: '120'
+      tst_cmdfiles: ""
+
+  kselftest-lib:
+    <<: *kselftest
+    params:
+      job_timeout: '360'
+      kselftest_collections: lib
+      tst_cmdfiles: "lib"
```

Comment:
Where is this used?

Reply:
this is not used in kselftest context, thanks :)